### ISSUE TITLE
Biometric device on USB Hub

### ISSUE THREAD

**Issue by SrujanMungara** [jeevareddy/rdservice#2]
When i'm trying to invoke capture services on USB hub, the device is getting reinitialised. But it is working good when connected directly to the device port. 

Can u please suggest a solution

**Comment by jeevareddy** [comment]
Hi @SrujanMungara,

Thanks for getting in touch here,

If you can provide the bellow information, it will be helpful for me to close this issue.
1. Device Make and Model that you are using for reading biometrics.
2. Device Make and Model of the Android device.
3. Fluttter Version and Channel

Please feel free to add any information that is relevant.

**Comment by SrujanMungara** [comment]
Hi Mr. Jeeva reddy,
Please find the details as requested

Biometric device- Mantra MFS 100,Startek FM 220
Android Device- Android 8, lenovo k8 plus and Redmi 10A- Android 10
flutter version-2.8.1

**Comment by jeevareddy** [comment]
Thanks for providing the details,

I will check if this issue can be reproduced.

**Comment by jeevareddy** [comment]
@SrujanMungara,

Currently, I don't have the Biometric device with me.

If you have an urgent requirement, please feel free to collaborate on this issue raising a pull request.

Thanks.

**Comment by ritesh685** [comment]
Hi @SrujanMungara  what do you mean by USB ,

you connected the device to the PC and try to get the device info from this plugin ?

**Comment by SrujanMungara** [comment]
No. i've connected the device with a usb hub to the Mobile Phone(Android)

**Comment by jeevareddy** [comment]
@SrujanMungara,
Are you facing the same problem with different biometric devices, with different USB cable,s and different Android Phones?

**Comment by SrujanMungara** [comment]
yes

**Comment by jeevareddy** [comment]
@SrujanMungara,
Do you still face this issue?
